RealestateApi/Repository/Interfaces/ICustomerRepository.cs

```diff
@@ -1,9 +1,12 @@
-﻿using RealEstateApi.Model;
+﻿using RealEstateApi.Dto.Request;
+using RealEstateApi.Dto.Response;
+using RealEstateApi.Model;
 
 namespace RealEstateApi.Repository.Interfaces
 {
     public interface ICustomerRepository
     {
         Task<List<ClientModel>> GetAllCustomersAsync();
+        Task<List<ClientModel>> AddCustomerAsync(CustomerRequestDto customerData);
```

Comment:
You are not returning a list because you are just returning one.
You should return just a ClientModel.

Reply:
Done. Returning only a Task<ClientModel>.